native/cocos/network/WebSocket-okhttp.cpp

```diff
@@ -372,7 +360,7 @@ JNI_PATH(nativeOnStringMessage)(JNIEnv * /*env*/,
                                 jlong handler) {
     auto *wsOkHttp3 = HANDLE_TO_WS_OKHTTP3(handler); // NOLINT(performance-no-int-to-ptr)
     std::string msgStr = JniHelper::jstring2string(msg);
-    RUN_IN_GAMETHREAD(wsOkHttp3->onStringMessage(msgStr));
+    wsOkHttp3->onStringMessage(msgStr);
```

Comment:
Why change it?

Reply:
> Why change it?

The Java side has ensured that nativeOnStringMessage runs on the game thread.

Reply:
I see. But i don't know why need to do synchronization in java instead in c++?

Reply:
While CocosActivity run on background, the old logic will create jni's global object. And the number of global object may exeed system's limit.

Reply:
The current implementation also use jni to invoke java codes. Why it doesn't have the problem?

Reply:
The global object is created during the nativeOnBinaryMessage call.